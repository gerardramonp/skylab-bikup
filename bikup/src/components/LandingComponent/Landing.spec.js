xit('should work', () => {
	expect(true).toBe(true);
});
