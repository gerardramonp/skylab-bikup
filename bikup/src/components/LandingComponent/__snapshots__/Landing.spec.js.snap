// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Component should match 1`] = `
<div
  className="landing"
>
  <div
    className="landing__header desktop"
  >
    <a
      aria-current="page"
      className="header__linkhome active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      <img
        alt="logo"
        className="logo"
        src="https://cdn.discordapp.com/attachments/692420285143711814/693437226146594876/LogoGerili.png"
      />
      <p
        className="appName"
      >
        bikUP
      </p>
    </a>
    <div
      className="flex-spacer"
    />
    <a
      aria-current={null}
      className="landing-header-login"
      href="/login"
      onClick={[Function]}
    >
      <button
        className="login__button login__button--header"
      >
        Log In
      </button>
    </a>
  </div>
  <div
    className="landing__upper mobile"
  >
    <a
      aria-current={null}
      className="login__button"
      href="/login"
      onClick={[Function]}
    >
      Log In
    </a>
  </div>
  <div
    className="landing__topContent"
  >
    <div
      className="topContent-main"
    >
      <img
        alt="bikUP logo"
        className="landing__bikup-logo"
        src="https://trello-attachments.s3.amazonaws.com/5f4cb639a6f5eb1005114de4/5f53aa4c97b74918797ef889/49939c4bd610049ad5bb132cd4fb6c87/bikup-logo.png"
      />
      <h1>
        bikUP
      </h1>
    </div>
    <div
      className="landing__separator"
    />
    <h2
      className="subtitle"
    >
      "Care for your bike"
    </h2>
    <div
      className="landing__register--container"
    >
      <div
        className="register__strava registerItem"
      >
        <button
          className="register__btn strava"
        >
          <a
            className="button__anchor button__anchor--strava"
            href="http://www.strava.com/oauth/authorize?client_id=37664&response_type=code&redirect_uri=http://localhost:3000/auth/strava&approval_prompt=force&scope=read_all,activity:read,profile:read_all"
          >
            <div
              className="btn__logo"
            >
              <img
                alt="google logo"
                src="https://trello-attachments.s3.amazonaws.com/5f4cb639a6f5eb1005114de4/5f53a18bf970328231db4f61/2f6af477333d585dfbfc19a14da9f857/strava-2.svg"
              />
            </div>
            <div
              className="btn__text"
            >
              Register with Strava
            </div>
          </a>
        </button>
      </div>
      <div
        className="register__google registerItem"
      >
        <button
          className="register__btn google"
        >
          <div
            className="btn__logo"
          >
            <img
              alt="google logo"
              src="https://trello-attachments.s3.amazonaws.com/5f4cb639a6f5eb1005114de4/5f53a18bf970328231db4f61/99c6b9ab576aa9e324980db26e5fc7c0/google-logo.png"
            />
          </div>
          <div
            className="btn__text"
          >
            Register with Google
          </div>
        </button>
      </div>
      <div
        className="register__separator"
      >
        <div
          className="line"
        />
        <span
          className="bold"
        >
          Or
        </span>
        <div
          className="line"
        />
      </div>
      <div
        className="register__mail registerItem"
      >
        <button
          className="register__btn mail"
          onClick={[Function]}
        >
          <div
            className="btn__logo"
          >
            <img
              alt="google logo"
              src="https://image.flaticon.com/icons/svg/561/561127.svg"
            />
          </div>
          <div
            className="btn__text"
          >
            Create an account
          </div>
        </button>
      </div>
    </div>
  </div>
  <div
    className="landing__team"
  >
    <h2
      className="team__title"
    >
      Meet the team
    </h2>
    <div
      className="team__members"
    >
      <div
        className="members__item"
      >
        <img
          alt="member img"
          className="member__img member__img--mou"
          src="https://trello-attachments.s3.amazonaws.com/5f4cb639a6f5eb1005114de4/5f53b05ec814da84c46afcea/f2f116196437dddbc62dc5dff303f654/mou.jfif"
        />
        <div
          className="member__info"
        >
          <p
            className="member__name"
          >
            Mourad Mounim
          </p>
          <p
            className="member__desc"
          >
            "The runner"
          </p>
        </div>
      </div>
      <div
        className="members__separator mobile"
      />
      <div
        className="members__item"
      >
        <img
          alt="member img"
          className="member__img"
          src="https://trello-attachments.s3.amazonaws.com/5f4cb639a6f5eb1005114de4/5f53b05ec814da84c46afcea/e58df585979e8a973502d3aedd008570/gerard.jfif"
        />
        <div
          className="member__info"
        >
          <p
            className="member__name"
          >
            Gerard Ramon
          </p>
          <p
            className="member__desc"
          >
            "The biker"
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="landing__footer"
  >
    <p>
      Copyright Â© bikUP
    </p>
  </div>
</div>
`;
